lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -1331,14 +1376,33 @@ ParseResult FIRStmtParser::parseExpImpl(Value &result, const Twine &message,
     // redundant definition of TOK_LPKEYWORD_PRIM which is needed to get
     // around a bug in the MSVC preprocessor to properly paste together the
     // tokens lp_##SPELLING.
-#define TOK_LPKEYWORD(SPELLING) case FIRToken::lp_##SPELLING:
```

Comment:
Need to check whether MSVC still requires this, and update comment (or code) depending on how that goes.